mlir/lib/Dialect/Tensor/IR/TensorOps.cpp

```diff
@@ -3158,19 +3158,23 @@ struct FoldStaticPadding : public OpRewritePattern<PadOp> {
 
     // Extract the static info from the high and low operands.
     SmallVector<int64_t> constOperandsLow;
+    llvm::SmallVector<Value> newLows;
```

Comment:
nit: You don't need the `llvm::`, there's a using declaration for this and some other common types in the mlir namespace. Also below.

Reply:
Done.